File: trail1/소수 판별기/prime-number-discriminator.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int N;
    scanf("%d", &N);
    bool flag = true;
    for (int i=2; i<N; i++) {
        if (N % i == 0) {flag = false; break;}
    }
    char res = flag==true ? 'P' : 'C';
    printf("%c", res);
    return 0;
}

File: trail1/소수 판별기/prime-number-discriminator.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        bool flag = true;
        for (int i=2; i<N; i++) {
            if (N % i == 0) {flag = false; break;}
        }
        char res = flag==true ? 'P' : 'C';
        Console.WriteLine(res);
    }
}
